docs/organizations/roles-and-permissions-for-organizations.md

```diff
@@ -96,6 +96,14 @@ The table below maps the GitHub Cloud and GitHub Enterprise roles to the corresp
       <td>No</td>
       <td class="yes">Yes</td>
     </tr>
+    <tr>
+      <td>Configure default settings for Git provider integration</td>
```

Comment:
A slightly shorter version:

```suggestion
      <td>Configure default Git provider integration settings</td>
```

Reply:
Thanks! I was not sure if "default Git provider integration settings" was readable! 😄